packages/prisma/src/index.ts

```diff
@@ -0,0 +1,27 @@
+import { PrismaPg } from '@prisma/adapter-pg'
```

Comment:
master prisma client setup file -> one place to optimize and import everywhere instead of installing it in all places and doing the same thing repeatedly